File: server/spec/factories/track_activities.rb
FactoryBot.define do
  factory :track_activity do
    name { "MyString" }
    short_name { "MyString" }
    date { "MyString" }
    duration { 1 }
    laps { 1 }
    state { "MyString" }
    type { "" }
    car_division { nil }
    circuit_variant { nil }
    race_weekend { nil }
  end
end

File: client/src/app/dummy/tp/races/08-race-tp-c2/sunday/track-activities.ts
// models
import {
  TrackActivity,
  TrackActivityState,
  TrackActivityType
} from './../../../../../shared/model/track-activity.model';

// dummy data
import { TP_C2_CAR_DIVISION } from './../../../car-divisions';
import { F08_TP_C2_CIRCUIT_VARIANT } from './../circuit-variants';

export const F08_TP_C2_SUNDAY_TRACK_ACTIVITIES: TrackActivity[] = [
  {
    id: '323',
    name: 'Final',
    short_name: 'FINAL',
    date: '2018-10-14T10:40:00.000-03:00',
    duration: 30,
    laps: 17,
    state: TrackActivityState.waiting,
    type: TrackActivityType.race,
    race_participants_track_activities: [],
    car_division: TP_C2_CAR_DIVISION,
    circuit_variant: F08_TP_C2_CIRCUIT_VARIANT
  }
];